modules/sdk/gradle-plugins-test-integration/src/main/java/com/liferay/gradle/plugins/test/integration/TestIntegrationPlugin.java

```diff
@@ -150,6 +159,34 @@ private static int _updateStartedAppServerStopCounters(
 		return originalCounter;
 	}
 
+	private Configuration _addConfigurationTestModules(final Project project) {
+		Configuration configuration = GradleUtil.addConfiguration(
+			project, TEST_MODULES_CONFIGURATION_NAME);
+
+		configuration.defaultDependencies(
+			new Action<DependencySet>() {
+
+				@Override
+				public void execute(DependencySet dependencySet) {
+					_addDependenciesTestModules(project);
+				}
+
+			});
+
+		configuration.setDescription(
+			"Configures additional OSGi modules to deploy during integration " +
+				"testing.");
+		configuration.setVisible(false);
+
+		return configuration;
+	}
+
+	private void _addDependenciesTestModules(Project project) {
+		GradleUtil.addDependency(
+			project, TEST_MODULES_CONFIGURATION_NAME, "org.apache.aries.jmx",
```

Comment:
Yeah, they're transitive dependencies. The `testModules` configuration is transitive by default.